src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js

```diff
@@ -210,14 +207,14 @@ export const GenericTable = ({
     }
     GenericTable.downloadLocked[lockId] = true;
     try {
-      if (!currentScenarioDatasetList || currentScenarioDatasetList?.length === 0)
+      const sourceDatasetId = context.targetDatasetId;
+      if (!sourceDatasetId)
```

Comment:
If we delete `currentScenarioDatasetList` from the component, _GenericTable_ as scenario parameter won't be able to get the id of its dataset because it doesn't have it in `context` :thinking: we need to keep `currentScenarioDatasetList` as fallback value, otherwise to pass it through `context` as well

Reply:
When used in the _scenario parameters_ panel, I think the context [already has](https://github.com/Cosmo-Tech/azure-sample-webapp/blob/main/src/components/ScenarioParameters/ScenarioParameters.js#L150) the scenario dataset as `targetDatasetId` (IIRC, it's been added in the context for the dynamic values of _enum_ and _multi-select_ components)